### PR TITLE
dcos-enterprise-cli: update minDcosReleaseVersion to 1.9

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: repo/packages/D/dcos-enterprise-cli/12/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "dcos-enterprise-cli",
   "version": "1.0.8",
   "maintainer": "support@mesosphere.io",
-  "minDcosReleaseVersion": "1.8",
+  "minDcosReleaseVersion": "1.9",
```
Comment (vespian) [inline repo/packages/D/dcos-enterprise-cli/12/package.json]: Aren't we worried that we are changing existing/already released package here? Also - why this change in general?

Comment (tamarrow) [inline repo/packages/D/dcos-enterprise-cli/12/package.json]: we have to update because if we add a new package that restricts to 1.9 only 1.8 clusters will still get package 12, which we don't want. 

the endpoint here only exists in 1.9+ (which is the reason for this change):  https://github.com/mesosphere/dcos-security-cli/blob/master/dscli/__init__.py#L62
